apps/system/js/app_window.js

```diff
@@ -1841,7 +1841,7 @@
           return;
         }
         invoked = true;
-        setTimeout(callback);
+        callback();
```

Comment:
Looking at the implementation of waitForNextPaint, it's possible that the callback can be called immediately (happens if either the browser or browser.element are null) - This is ok if this can't happen in ready (which I guess it can't?), but otherwise there's the chance the callback will execute before the code below.

I'd say that perhaps it makes sense to remove this setTimeout, but to add one again in browser_mixin.js's implementation of waitForNextPaint.